### DISCUSSION TITLE
Annual Meeting 2022

### DISCUSSION THREAD

**Discussion by gregmacfarlane** [tfresource/tfresource.github.io#373] [General]
**Minutes to come** Later:

Attendees:

**Comment by gregmacfarlane** [comment]
Greg Macfarlane was here, Dave Ory left early.

**Reply by ClarissaLiv** [reply]
I got here late, sorry

**Reply by nkuehnel** [reply]
I was here

**Reply by peter-c12** [reply]
I was here and would like to be involved with the group, attending meeting, etc.

**Reply by zhuojunj** [reply]
I was here (Zhuojun Jiang, zhuojun.jiang@dot.ohio.gov). Thanks!
